lib/measures/tbd/README.md

```diff
@@ -44,21 +36,29 @@ Loads existing 'tbd.json' file (under '/files'), may override 'default thermal b
 **Model Dependent:** false
 
 ### Default thermal bridge set
-e.g. 'poor', 'regular', 'efficient', 'code' (may be overridden by 'tbd.json' file).
+e.g. '90.1.22|steel.m|unmitigated' (may be overridden by 'tbd.json' file).
 **Name:** option,
 **Type:** Choice,
 **Units:** ,
 **Required:** false,
 **Model Dependent:** false
 
 ### Write 'tbd.out.json'
-Write out 'tbd.out.json' file e.g., to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
+Write out 'tbd.out.json' file, e.g. to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
 **Name:** write_tbd_json,
 **Type:** Boolean,
 **Units:** ,
 **Required:** false,
 **Model Dependent:** false
 
+### Wall-roof edge as 'parapet'
+Leave CHECKED if wall-roof edge is considered a parapet or an overhang (see ASHRAE 90.1 2022 5.5.5.1 & A10).
+**Name:** parapet,
+**Type:** Boolean,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
```

Comment:
New measure argument, namely in support of 90.1 2022 A10: a switch from more common _parapet_ (wall-roof intersection) to _roof_ (other than _parapet_ or _overhang_). Optional, and defaults to `true` (i.e. "consider as parapet"). Built-in PSI sets have now both _parapet_ and _roof_ edge types; users can opt for one or the other.